packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -340,7 +340,13 @@ export async function launch({
   }
 
   async function installBundle(bundleSource) {
-    const bundle = JSON.parse(bundleSource);
+    let bundle;
+    try {
```

Comment:
I wonder if there should be a try/catch around the FFI connection between golang and JS. golang clearly doesn't want to hear about any problems from JS.

For example, are we sure we want this to crash the node?

```js
      default: {
        assert.fail(X`${action.type} not recognized`);
      }
```

But I suppose that's a separable issue.